### ISSUE TITLE
Perform classification experiment

### ISSUE THREAD

**Issue by JankowskiDaniel** [JankowskiDaniel/Neural-Deep-Retina#61]
* Simplify the problem to the classification instead of regression. Apply a pre-defined threshold value to determine whether the output is 1.0 (peak) or 0.0 (no peak).
* Apply a proper loss function to the classification problem
* Fine-tune such obtained model for the regression task
